stable/hubot/README.md

```diff
@@ -47,26 +47,38 @@ The following table lists the configurable parameters of the Hubot chart and the
 
 Parameter | Description | Default
 --- | --- | ---
-`namespace` | namespace to use | `default`
-`replicaCount` | desired number of pods | `1`
-`image.repository` | container image repository | `minddocdev/hubot`
-`image.tag` | container image tag | `latest`
-`image.pullPolicy` | container image pull policy | `Always`
-`service.type` | type of service to create | `NodePort`
-`service.httpPort` | port for the http service | `80`
+`fullnameOverride` | Override the full resource names | `""`
```

Comment:
This is the attribute of any good `_helper.tpl`. It allows getting rid of duplicity in objects names. For example, if the release name is `hubot` and `fullnameOverride` is not set, you will get all objects named `hubot-hubot-`.

With `fullnameOverride=hubot` it will be just `hubot`. Let's keep this.